src/main/java/org/sonar/plugins/web/checks/structure/ChildElementIllegalCheck.java

```diff
@@ -58,7 +58,10 @@ public void startElement(TagNode element) {
           createViolation(childNode.getStartLinePosition(), "Remove this \"" + child + "\" tag; it is not a legal child of \"" + parent + "\".");
         }
       }
+    } else if (element.equalsElementName(parent)) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) This branch can not be reached because the condition duplicates a previous condition in the same sequence of "if/else if" statements [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1862)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)